src/constrain_pot.f90

```diff
@@ -357,10 +357,10 @@ subroutine constrain_hess(fix,n,at,xyz0,Hess)
          ii = (ia-1)*3+ic
          xyz(ic,ia)=xyz0(ic,ia)+step
          gr=0.0_wp
-         call constrpot(fix,n,at,xyz,gr,e)
+         call constrpot(n,at,xyz,gr,e)
          xyz(ic,ia)=xyz0(ic,ia)-step
          gl=0.0_wp
-         call constrpot(fix,n,at,xyz,gl,e)
+         call constrpot(n,at,xyz,gl,e)
```

Comment:
This part probably contains dead code, nevertheless, with this refactor the procedure call uses the correct interface.